notify-slack-deploy/notify.py

```diff
@@ -10,8 +10,13 @@
 workflow = github["workflow"]
 repository = github["repository"]
 run_id = github["run_id"]
-application = github["event"]["repository"]["name"]
-html_url = github["event"]["repository"]["html_url"]
+
+# In scheduled events, github["event"] is just the cron string
+if isinstance(github["event"], dict) and "repository" in github["event"]:
+    html_url = github["event"]["repository"]["html_url"]
+else:
+    html_url = f"https://www.github.com/{repository}"
```

Comment:
Given that this case works for all events, do we need the other branch of the if?

Reply:
Probably not!